FusionMCPBridge/handlers/design/geometry_impl2.py

```
@@ -0,0 +1,642 @@
+            ui.messageBox("Keine Bodies gefunden.")
+            return
+        
+        inputEntites = adsk.core.ObjectCollection.create()
+        inputEntites.add(latest_body)
+        
+        if plane == "XY":
+            sketch = sketches.add(rootComp.xYConstructionPlane)
+        elif plane == "XZ":
+            sketch = sketches.add(rootComp.xZConstructionPlane)
+        elif plane == "YZ":
+            sketch = sketches.add(rootComp.yZConstructionPlane)
```

Ruff (0.14.10)

Local variable `sketch` is assigned to but never used

Remove assignment to unused variable `sketch`

(F841)